pages/common/eslint.md

```diff
@@ -9,7 +9,7 @@
 
 - Lint on a given set of files:
 
-`eslint {{filename}}.js {{filename1}}.js`
+`eslint {{path/to/file}}.js {{path/to/file1}}.js`
```

Comment:
> Is it just me or is that not making remotely any sense?

It is technically right and it would work. But it looks odd TBH. Also, I am not sure on the previous page (esbuild) whether there must be a path with `index.js`.